src/store/account-details/account-details.thunks.ts

```diff
@@ -243,13 +262,22 @@ function fetchExits(tokenId: Token["id"]): AppThunk {
           dispatch(globalThunks.recoverPendingDelayedWithdrawals(exits));
           dispatch(accountDetailsActions.loadExitsSuccess(exits));
         })
-        .catch((err: unknown) =>
+        .catch((error: unknown) => {
+          const errorMsg = adapters.parseError(
+            error,
+            "An error occurred on src/store/account-details/account-details.thunks.ts:fetchExits"
```

Comment:
I don't think that this is a good idea. What if we rename a file or if we move it to another place?

In fact, this second parameter doesn't provide any relevant information and I would delete it from this`adapters.parseError` function. What if you write this string wrong? A copy and paste for example? Sending `unknown error: JSON.stringify(error)` in this case, which is very unlikely to happen, is more than enough IMO.

Reply:
Yes, you're right, this requires maintenance.

> doesn't provide any relevant information

I think it provides relevant information. When you are faced with an error from a customer on production, how will you know on what module and function the error originated? Don't you think this is useful data? Or maybe we can get that info from somewhere else I'm not counting on?

Reply:
It provides relevant information only:

- If you write the path and the name of the function right.
- If you don't modify the name of the function or of the file or even if you don't move the file in the future.

We can not get this info from somewhere as this info is only in the `Error` class (`error.stack`). But if this is not supported by the language we cannot go too far, maintaining this strings is going to be a hell compared to the useful info that it's going to provide.

Reply:
Since we're removing the function `parseError` we still need to decide what do we want to store in the state for each errored AsyncTask.

Reply:
We agreed on keeping the function `parseError`:

  * Without `prefixMsg` 
  * Without parsing the decoding issues to a human-readable string
  * Including the serialized error with the stack trace

Reply:
Done